src/LinuxCaptureService/LinuxCaptureService.cpp

```diff
@@ -198,9 +200,28 @@ grpc::Status LinuxCaptureService::Capture(
     } else {
       FilterOutInstrumentedFunctionsFromCaptureOptions(
           result_or_error.value().instrumented_function_ids, linux_tracing_capture_options);
-      LOG("User space instrumentation enabled for %u out of %u instrumented functions.",
+
+      const std::string status_message = absl::StrFormat(
+          "User space instrumentation enabled for %u out of %u instrumented functions.",
           result_or_error.value().instrumented_function_ids.size(),
           capture_options.instrumented_functions_size());
+      LOG("%s", status_message);
+
+      // If at least one function was not instrumented we send a message to the capture log.
```

Comment:
What is the relationship with the `if (result_or_error.has_error()) {` case? In that case we also fall back to uprobes (right?), just like if no function could be instrumented in this branch. I'm not sure one case can be included in the other, but we should handle the overlap, for example warn about falling back to uprobes also in the `if (result_or_error.has_error()) {` case.